app/config.ts

```diff
@@ -2,7 +2,16 @@ declare global {
   const CONFIG: any;
 }
 
-const config = {
+interface Config {
+  LINKSF_DOMAIN: string;
+  MOHCD_DOMAIN: string;
+  MOHCD_SUBDOMAIN: string;
+  SFFAMILIES_DOMAIN: string;
+  UCSF_DOMAIN: string;
+  [key: string]: any;
```

Comment:
It seems like all of these values are probably strings, but I didn't want to make that assumption considering some of them are passed in dynamically by the app's ENV object (though I guess all env args are probably strings by default??)

Reply:
Yeah, I think these will probably always be strings in practice, but it is hard to be 100% type safe here because these all ultimately come from a Webpack plugin that injects these values in at build time.

Another alternative to `any` is [`unknown`]([url](https://mariusschulz.com/blog/the-unknown-type-in-typescript)). Whereas `any` will always pass all type checks, `unknown` basically won't let you do anything with it unless you either use a `typeof` expression, a custom type guard, or make a type assertion. Although this may cause more friction to a developer, it might serve as a reminder for us to go back and update this interface with the precise type.

Reply:
Awesome, that's a great suggestion. Thanks for the feedback.